include/golioth/client.h

```diff
@@ -97,18 +97,18 @@ struct golioth_pki_credential {
     size_t private_key_len;
 };
 
-/// TLS Authentication Credentials
-typedef struct {
+/// TLS Authentication Credential
+struct golioth_credential {
```

Comment:
Same comment on mentioning the name change (dropped `s`) reasoning in the commit message.